src/runtime/ContextFeed.tsx

```diff
@@ -44,7 +44,14 @@ class ContextFeed extends React.Component<Props>
           )
         } else {
           if (values.indexOf(undefined) > -1) {
-            return <Spinner name='double-bounce' />
+            // Hacky way to find index of ComponentLogs in values
+            // TODO: find a better way to identify type of object
+            let i = _.findIndex(values, function(o: any) {
```

Comment:
So here it looks like you're searching for the [`_react`](https://github.com/dOrgTech/DAOcomponents/blob/d42f18d3c2d3b4414abb1e12e4dd15c7e3865acb/src/runtime/logging/ComponentLogs.ts#L61) member of the `ComponentLogs` class, but this won't work in all cases. Let me explain.

`values` in this function refers to the array of context consumers that're being consumed. For example in this case:
```html
<DAO.Data>
<Member.Data>
<Member.Logs>
{(dao, member, logs) => ( ... )}
...
```
`values` would contain [dao.data, member.data, member.logs], and the value of the index `i` would be `2`.

This breaks down however when `<Member.Logs>` is not present. For example:
```html
<DAO.Data>
<Member.Data>
{(dao, member) => ( ... )}
...
```
The value of the index `i` in your code would be -1.

So then what's the desired behavior and the solution to get to this?

In the case of the above code sample, if `<Member.Data>` is undefined, we want to show a spinner letting us know that the loading of the `Member` component is still in progress. In order to do this, we need to somehow know that the `Data` consumer is coming from `Member`, and then get the `Member.Logs` consumer... I'm thinking this requires a refactor of some sort. I'll look into this now and see if a solution pops up...